cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -5369,6 +5369,14 @@
     </description>
   </property>
 
+  <property>
+    <name>app.version.deletion.enabled</name>
+    <value>false</value>
```

Comment:
this should also default to true so that the behavior is the same as before when somebody upgrades.

Reply:
same comment as above